packages/hash/frontend/src/pages/invite.page.tsx

```diff
@@ -1,11 +1,10 @@
-import { NextPage } from "next";
 import { tw } from "twind";
 import { useRouter } from "next/router";
 import { useMutation } from "@apollo/client";
 import { useCallback, useEffect, useMemo, useState } from "react";
 import { useUser } from "../components/hooks/useUser";
 
-import { AuthLayout } from "../components/layout/PageLayout/AuthLayout";
+import { AuthLayout } from "./shared/auth-layout";
```

Comment:
It's a bit confusing to see a layout not amongst other layouts from the `../shared/layouts` ‘lib’. We can untangle this in a follow-up PR

Reply:
Fixed in [8c19e64](https://github.com/hashintel/hash/pull/468/commits/8c19e64f44905c778cdeca75b7c8d919b5e779b3)

Reply:
We still have a separate ‘lib’ with auth layout, but let's leave it for now. We can consider making it a sibling of other layouts or stop calling it a layout. This will help us avoid two collections of layouts.